docs/_pages/papers.md

```diff
@@ -46,6 +46,7 @@ We only add paper to this list before we decide to oral/poster it at our AMC sem
 
 | Seminar | Member | Paper |
 | ------- | ------ | ----- |
+| [S3E12](https://ai-ml.club/events/seminar-meeting-minutes-3-12/) | [@qhduan](https://github.com/qhduan) | [Deep Biaffine Attention for Neural Dependency Parsing](https://arxiv.org/abs/1611.01734) |
```

Comment:
```diff
- S3E12
+ S3E13
```

Reply:
@CyFeng16 Notice: the url of the blog page hasn't been changed. (from a passerby bug hunter)

Reply:
@hwfan You could report the problem and form a single issue looking for help.